Modules/ThirdParty/HDF5/src/itkhdf5/src/itk_hdf5_mangle.h.in

```diff
@@ -3847,258 +3943,1530 @@ e.g. remove mangling for H5Fget_info but leave it for H5Fget_info1 and H5Fget_in
 #define H5_timer_start @MANGLE_PREFIX@_H5_timer_start
 #define H5_timer_stop @MANGLE_PREFIX@_H5_timer_stop
 #define H5_trace @MANGLE_PREFIX@_H5_trace
+#define H5_trace_args @MANGLE_PREFIX@_H5_trace_args
 #define H5_type_conv_blk_free_list @MANGLE_PREFIX@_H5_type_conv_blk_free_list
 #define H5_vlen_fl_buf_blk_free_list @MANGLE_PREFIX@_H5_vlen_fl_buf_blk_free_list
 #define H5_vlen_vl_buf_blk_free_list @MANGLE_PREFIX@_H5_vlen_vl_buf_blk_free_list
 #define H5allocate_memory @MANGLE_PREFIX@_H5allocate_memory
+#define H5atclose @MANGLE_PREFIX@_H5atclose
 #define H5check_version @MANGLE_PREFIX@_H5check_version
 #define H5close @MANGLE_PREFIX@_H5close
-#define H5dontAtexit_called @MANGLE_PREFIX@_H5dontAtexit_called
 #define H5dont_atexit @MANGLE_PREFIX@_H5dont_atexit
 #define H5free_memory @MANGLE_PREFIX@_H5free_memory
 #define H5garbage_collect @MANGLE_PREFIX@_H5garbage_collect
-#define H5get_alloc_stats @MANGLE_PREFIX@_H5get_alloc_stats
 #define H5get_free_list_sizes @MANGLE_PREFIX@_H5get_free_list_sizes
 #define H5get_libversion @MANGLE_PREFIX@_H5get_libversion
+#define H5is_library_terminating @MANGLE_PREFIX@_H5is_library_terminating
 #define H5is_library_threadsafe @MANGLE_PREFIX@_H5is_library_threadsafe
 #define H5libhdf5_settings @MANGLE_PREFIX@_H5libhdf5_settings
 #define H5open @MANGLE_PREFIX@_H5open
 #define H5resize_memory @MANGLE_PREFIX@_H5resize_memory
 #define H5set_free_list_limits @MANGLE_PREFIX@_H5set_free_list_limits
 #define HDrand @MANGLE_PREFIX@_HDrand
 #define HDsrand @MANGLE_PREFIX@_HDsrand
-#define IEEE_F32BE @MANGLE_PREFIX@_IEEE_F32BE
```

Comment:
Removing definitions like this can cause problems for someone else.

Reply:
Indeed. The mangling is WIP. We'll also need to run the commands on Windows & linux.